docs/ref/models/querysets.txt

```diff
@@ -979,6 +979,16 @@ database.
     performance, since you have done a database query that you haven't used. So
     use this feature with caution!
 
+.. versionchanged:: 1.11
```

Comment:
Yes, the idea is to mention the behavior twice. Once in the section body and once at the bottom under the versionchanged annotation. When removing the annotation, we remove the annotation and the note below it, leaving the body text.